src/components/Navbar.test.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/jsx-closing-tag-location */
```

Comment:
- [ ] Please kindly fix your linters errors instead of disabling them. Fix linters help us to write clean and professional codebase 😉